### DISCUSSION TITLE
Spent Hours Just Trying to Theme to KDE Applications inside of i3WM

### DISCUSSION THREAD

**Discussion by VehementHam** [i3/i3#5539] [Q&A] [answered]
I am stuck on this. All I want to do is theme two applications. Krusader (native) and Haruna (flatpak). Theming engines are a PITA. I know that KDE applications use `qt5ct` or `qt6ct`. I have tried both. I can't figure out how to apply this. Here is the theme that I want to use: https://store.kde.org/p/1327093. I have tried installing it form that page, installing it from the github, and using the entire folder as well as the sub folders. I have placed them in ever directory that could POSSIBLY be read by `qt5ct` or `qt6ct`. Those directories include `~/.themes`, `~/.local/share/plasma/desktoptheme/` and `~/.config/qt6ct/colors/`. Here is my `/etc/environment`:
```
#
# This file is parsed by pam_env module
#
# Syntax: simple "KEY=VAL" pairs on separate lines
#
QT_QPA_PLATFORMTHEME="qt6ct"

QT_PLATFORMTHEME="qt6ct"

QT_PLATFORM_PLUGIN="qt6ct"

QT_AUTO_SCREEN_SCALE_FACTOR=0
```
The issue is that when I open qt6ct, it does not show the Nordic theme as an option in "style".
I have gotten to the point where I cannot figure this out on my own and need hep from the experts.
Here is a troubleshooting conversation I had with ChatGPT: https://chat.openai.com/c/d5735624-2727-49eb-b843-386327794194
I have also read every forum post I could find.

**Comment by yusufaktepe** [comment]
qtXct applications allow you to change the theme engine, not the theme.

- Install [kvantum](https://github.com/tsujan/Kvantum/tree/master/Kvantum)
- Set style to kvantum in qtXct
- Set theme to [nordic](https://github.com/EliverLara/Nordic) in Kvantum Manager.

Hope this helps.

**Reply by VehementHam** [reply]
How do I set the Nordic theme in Kvantum Manager? It does not appear as an option.

**Reply by yusufaktepe** [reply, marked as answer]
Copy [these](https://github.com/EliverLara/Nordic/tree/master/kde/kvantum) folders to `~/.config/Kvantum` and it should show up in Kvantum Manager like this:
![Screenshot_2023-06-24_23-50-37](https://github.com/i3/i3/assets/32964025/695e76f2-f7d3-4b3f-bad4-b5f82a4a1acd)

See [here](https://github.com/tsujan/Kvantum/blob/master/Kvantum/INSTALL.md#notes-for-theme-makers).

**Reply by VehementHam** [reply]
Thank you! I love you, bro. You are the best :)

**Reply by VehementHam** [reply]
Still not working with the flatpak, but I am sure that it should be a much easier fix. It works with Krusader.
